src/duration.rs

```diff
@@ -310,7 +356,10 @@ impl Add for Duration {
             nanos -= NANOS_PER_SEC;
             secs += 1;
         }
-        Duration { secs: secs, nanos: nanos }
+        Duration {
+            secs: secs,
+            nanos: nanos,
+        }
```

Comment:
It looks like there's a lot of formatting-related changes in this diff, can those all be removed into a separate PR?